### ISSUE TITLE
Avoid adding alt text field if there's no alt text

### ISSUE THREAD

**Issue by keerifox** [Lexevolution/reso-announce#2]
it can be misleading and accessibility unfriendly to mark media as containing alt text when it has none, as it makes the screen readers spell it out and may confuse creatures that don't wish to share images without alt text into thinking there is one

it is preferred to omit the field in such cases

https://github.com/Lexevolution/reso-announce/blob/3d2d72d973f8f529d547c1b38be563b5d9b6077a/index.mjs#L47

appreciate the alt text feature otherwise, and the relay as a whole

@jae1911

**Comment by Lexevolution** [comment]
No need, I have commited the change :) I'll deploy it tomorrow (my time UTC+10)
